java/google/registry/request/BUILD

```diff
@@ -21,15 +19,15 @@ java_library(
         "@com_googlecode_json_simple",
         "@javax_servlet_api",
         "@joda_time",
+        "//java/google/registry/security",
```

Comment:
Most of the diffs in this file are unnecessary.  We can't reorder imports anyway; their sorting is strictly enforced by a presubmit check on our side.  So we'll add the new one but the reordering changes won't get made.